### ISSUE TITLE
Chatlog not working

### ISSUE THREAD

**Issue by RagingWookie** [grimdesigns-fivem/FiveM-Discord-Bot#12]
I am unable to get my Chatlog working.  System log and Kill Log is working 100%.  If someone types anything into chat, it doesn't go through to the Webhook.  This worked up until the latest FiveM updates and then stopped working.  I did not change scripting.

**Comment by RagingWookie** [comment]
I have even tried configuring my own commands in the config file, they also don't go through.  These are standard commands such as:
/car
/cardel
/revive
/heal
/tp
/giveitem

**Comment by JakubLudvik** [comment]
i have got same problem

**Comment by GrimDesignsFiveM** [comment]
@RagingWookie @JakubLudvik This script is extremely outdated. I will be updating it soon though 👌 in the meantime I will look for a fix for you guys 😁

You have all your webhooks set up?

**Comment by RagingWookie** [comment]
Yeah, webhooks already set up.  Your help will mean a lot buddy.  Thanks :)

**Comment by GrimDesignsFiveM** [comment]
@RagingWookie Do you have discord?

**Comment by GrimDesignsFiveM** [comment]
### **Required Updates:**
-------


- [ ] Fix RP Commands Not Working With Newest CFX Version

- [ ] Push Updates

- [ ] Release Changes

- [ ] Update to v1.5.2
-------


Follow this post for status updates on a fix for this issue
-------

**Comment by GrimDesignsFiveM** [comment]
### **Required Updates:**
-------


- [ ] Fix RP Commands Not Working With Newest CFX Version

- [ ] Push Updates

- [ ] Release Changes

- [ ] Update to v1.5.2
-------


Follow this post for status updates on a fix for this issue
-------

### **Contact Options:**

- [My Twitter](https://www.twitter.com/TheRealToxicDev)

- [My Discord](https://discord.io/Toxic-Team)

-------
